### PR TITLE
CART-881 swim: delay the first ping until all will be initialized

### PR DESCRIPTION
**Pull request by dmiter**
- delay the first ping until all things will be initialized
- check about SWIM inited from few public API
- disable SWIM in crt_launch

### CODE REVIEW

**Comment by dmiter** [comment]
Test fails because of DAOS-3945 - DaosCoreTest Failed to stop servers

**Comment by jolivier23** [comment]
There are a couple of test failures I haven't seen in any recent pages.   BadEvict and ior_small are probably known issues.  Can you check the others?

**Comment by dmiter** [comment]
DAOS-4466 - IorSmall.1-./io/ior_small.py:IorSmall.test_ior_small test failing with setup issues.
DAOS-4410 - pool EvictTests times out with BAD_SERVER_NAME

File: src/cart/src/cart/crt_swim.c [hunk 1]
```diff
@@ -539,6 +541,8 @@ int crt_swim_enable(struct crt_grp_priv *grp_priv, int crt_ctx_idx)
 	if (old_ctx_idx != -1) {
 		rc = crt_unregister_progress_cb(crt_swim_progress_cb,
 						old_ctx_idx, NULL);
+		if (rc == DER_NONEXIST)
```
Comment (ashleypittman) [inline src/cart/src/cart/crt_swim.c:544]: This should be -DER... shouldn't it?

Comment (dmiter) [reply src/cart/src/cart/crt_swim.c:544]: Yes, good point! Thanks!

**Pull request merged**